README.md

```diff
@@ -266,11 +276,11 @@ For example:
 
     oxview --log-path=/var/log/blueox --type-name="request" --start-date=20120313 --end-date=20120315
 
-Where `request` is the channel you want to examine.
+Where `request` is the event type you're interested in.
 
 You can also connect to `oxd` and get a live streaming of log data:
 
-    oxview -H localhost:3513 --type-name="request*"
+    oxview -H log-master --type-name="request*"
```

Comment:
this is a bit confusing when it comes to documentation since I might assume that `log-master` might has some special purpose like a reserved/keyword